WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionSheetViewController.swift

```diff
@@ -0,0 +1,277 @@
+import UIKit
+
+struct ReaderSortingActionSheetOption {
+    let title: String
+    let image: UIImage
+    let checked: Bool
+    let identifier: String
+    let sortingOption: ReaderSortingOption
+
+    init(title: String, image: UIImage, checked: Bool, identifier: String, sortingOption: ReaderSortingOption) {
+        self.title = title
+        self.image = image
+        self.checked = checked
+        self.identifier = identifier
+        self.sortingOption = sortingOption
+    }
+}
+
+class ReaderSortingActionSheetOptionControl: ClosureControl {
+    enum Constants {
+        static let iconsHeight: CGFloat = 24.0
+        static let iconsWidth: CGFloat = 24.0
+        static let iconLeading: CGFloat = 16.0
+        static let labelLeading: CGFloat = 6.0
+        static let minimalTappableHeight: CGFloat = 44.0
+        // TODO: check if we should color from assets or from hex
+        static let checkTintColor: UIColor = UIColor(light: UIColor(named: "Blue50") ?? UIColor(hexString: "2271B1"), dark: UIColor(named: "Blue30") ?? UIColor(hexString: "5198D9"))
```

Comment:
In Figma it says to use Blue50 and Blue30 for colors, but since using `UIColor(named:)` returns optional color I had to use hex colors as a fallback.